tests/p2p/test_split_brain.py

```diff
@@ -37,14 +38,12 @@ def create_peer(self, network, unlock_wallet=True):
         return manager
 
     @pytest.mark.slow
-    def test_split_brain_plain(self):
+    def test_split_brain_plain(self) -> None:
         debug_pdf = False
 
         manager1 = self.create_peer(self.network, unlock_wallet=True)
-        manager1.avg_time_between_blocks = 3
 
         manager2 = self.create_peer(self.network, unlock_wallet=True)
-        manager2.avg_time_between_blocks = 3
```

Comment:
Why are you changing this?

Reply:
After this test was typed, mypy complained that `HathorManager` does not have a `avg_time_between_blocks` property. Therefore, this was useless (probably an artifact from some older code) and removing it doesn't break the test.

Reply:
This line was commented instead. So, we are going to fix it later.